lib/middleware/user-context.js

```diff
@@ -21,10 +21,12 @@ module.exports = function userContextMiddleware() {
     loopbackContext.set('accessToken', req.accessToken.id);
     const app = req.app;
     const UserModel = app.accessUtils.options.userModel || 'User';
+    const TeamModel = app.accessUtils.options.groupAccessModel || 'Team';
```

Comment:
I'd prefer to use the name `GroupAccessModel` to be consistent with the rest of the codebase. 'Team' is just what it's called in the test data.